File: runtime/Jlib/custom.java
package bigloo;

public class custom extends obj
{
  private static custom nil = new custom();
   
  public byte[] identifier;
  
  public static custom nil() {
     return nil;
  }
   
  public boolean equal( final custom  o )
  {
    return (o == this);
  }

  public int hash()
  {
    return hashCode();
  }
}

File: runtime/CSlib/custom.cs
namespace bigloo
{
  public sealed class custom: obj
  {
    private static custom _nil = new custom();
    public byte[] identifier;

    public static custom nil() {
      return _nil;
    }
   
    public bool equal( custom  o )
    {
      return (o == this);
    }

    public int hash()
    {
      return GetHashCode();
    }
  }
}
